cwltool/main.py

```diff
@@ -132,6 +132,10 @@ def _terminate_processes() -> None:
                         p.kill()
             except FileNotFoundError:
                 pass
+        if process.stdin:
+            process.stdin.close()
+        process.wait(10)
```

Comment:
My concern is that this wait may raise a [`TimeoutExpired`](https://docs.python.org/3/library/subprocess.html#subprocess.Popen.wait) exception if the process doesn't end within 10s for some reason, which would interrupt the rest of the cleanup.

Reply:
Good point, I'll add a try/except